Doc/library/sqlite3.rst

```diff
@@ -1369,10 +1379,18 @@ This is how SQLite types are converted to Python types by default:
 +-------------+----------------------------------------------+
 
 The type system of the :mod:`sqlite3` module is extensible in two ways: you can
-store additional Python types in an SQLite database via object adaptation, and
-you can let the :mod:`sqlite3` module convert SQLite types to different Python
-types via converters.
+store additional Python types in an SQLite database via
+:ref:`object adapters <sqlite3-adapters>`,
+and you can let the :mod:`sqlite3` module convert SQLite types to
```

Comment:
Just a minor nit: the module is already linked earlier in the sentence, so this second occurrence could be converted to ` ``sqlite3`` `.  I'm not sure if this convention is followed throughout the document (I saw at least another ` ``sqlite3`` `), and the style guide doesn't seem to mention this.

If the two versions render differently, it might be better to use `:mod:` or just ` ``sqlite3`` ` for both, given that in this case the links points to the document itself and they are not particularly useful.

Reply:
I was going to point this out too; it just slipped my mind for some reason. It seems the convention @erlend-aasland has followed and advocated for (as discussed in the review feedback of one of his other recent PRs) is to just always use regular literals for the module name inside the same module.

The rendering for both (at least under the current theme) is as a literal, just with the `:mod:` role version also being a link (which isn't necessary here), though at least for this particular case it is moot since the convention seems to be just going with a verbatim literal including on first use.

Reply:
```suggestion
and you can let the ``sqlite3`` module convert SQLite types to
```